File: MATLAB/fuzzbuzz/fuzzbuzz.m
disp('A Fuzzbuzz Algorithm');

disp('Input A Number:');
num = input('');

if isnumeric(num)
    if mod(num,5) == 0 &&  mod(num,3) == 0
        disp('FUZZBUZZ')
    elseif mod(num,5) == 0
        disp('BUZZ');
    elseif mod(num,3) == 0
        disp('FUZZ');
    else
        disp('Not A Number Divisible By 3 and 5');
    end
else
    disp('Not A Number');
end

File: PHP/fuzzbuzz/fuzzbuzz.php
<?php
/**
 * Created by PhpStorm.
 * User: Doraemon
 * Date: 11/27/2018
 * Time: 11:50 AM
 */


makeFuzzBuzz("ANT");


function makeFuzzBuzz($number){
    if(is_numeric($number)){
        if($number % 3 == 0 && $number % 5 == 0)
            print_r("FUZZBUZZ");
        elseif($number % 3 == 0)
            print_r("FUZZ");
        elseif($number % 5 ==  0)
            print_r("BUZZ");
        else
            print_r("Not Divisible by 3 or 5");
    }else
        print_r("Not A Number");
}